File: Prime number-1/src/PrimeNumber.java
import java.util.Scanner;

public class PrimeNumber {

	public static void main(String[] args) {
		
		System.out.println("Enter limit : ");
		Scanner s=new Scanner(System.in);
		
		int limit=s.nextInt();
		int flag=0;
		
		for(int i=2;i<=limit/2;i++) {
			if(limit%i==0){
				flag=1;
				break;
			}
		}
		if(flag==1) {
			System.out.println("Number is not a prime number");
		}else {
			System.out.println("Number is a prime number");
		}
	}
}

File: Prime number/src/Prime number.c
/*
 ============================================================================
 Name        : Prime.c
 Author      : 
 Version     :
 Copyright   : Your copyright notice
 Description : Hello World in C, Ansi-style
 ============================================================================
 */

#include <stdio.h>
#include <stdlib.h>

int main(void) {
	int num,i,flag=0;
	setbuf(stdout,NULL);
	printf("Enter a number : ");
	scanf("%d",&num);
	for(i=2;i<=num/2;i++){
	if(num%i==0){
		flag=1;
		break;
	}
	}
	if(flag==0){
		printf("Number is prime number");
	}else {
		printf("Number is not a prime number");
	}
	return EXIT_SUCCESS;
}
